### PR TITLE
Unify link components

### PR DESCRIPTION
**Pull request by ravinalamada**
Hi all,

Changes for the unify-link-component branch are made in this branch. Please, have a look at it ([issue  #3633](https://github.com/trezor/trezor-suite/issues/3633)).

### CODE REVIEW

File: packages/suite/src/components/settings/SettingsMenu/index.tsx [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-import { Translation, ExternalLink, AppNavigationPanel, AppNavigation } from '@suite-components';
+import { Translation, TrezorLink, AppNavigationPanel, AppNavigation } from '@suite-components';
 import { useActions, useSelector } from '@suite-hooks';
 import * as modalActions from '@suite-actions/modalActions';
 import * as suiteActions from '@suite-actions/suiteActions';
 import * as routerActions from '@suite-actions/routerActions';
 import { SUPPORT_URL } from '@suite-constants/urls';
 
-const StyledLink = styled(ExternalLink)`
+const StyledLink = styled(TrezorLink)`
```
Comment (tomasklim) [inline packages/suite/src/components/settings/SettingsMenu/index.tsx:11]: It should not be underlined.
Screenshot 2021-09-07 at 14 27 58
Screenshot 2021-09-07 at 14 28 04

File: packages/suite/src/components/suite/modals/AddAccount/components/AccountTypeDescription.tsx [hunk 2]
```diff
@@ -50,9 +50,9 @@ export const AccountTypeDescription = ({ network, accountTypes }: Props) => {
     return (
         <Info size="small" textAlign="left">
             <Translation id={description} />{' '}
-            <ExternalLink href={url} size="small">
+            <TrezorLink href={url} size="small">
```
Comment (tomasklim) [inline packages/suite/src/components/suite/modals/AddAccount/components/AccountTypeDescription.tsx]: Icon is missing in your implementation

Screenshot 2021-09-07 at 14 31 37
Screenshot 2021-09-07 at 14 31 41

**Review by tomasklim — CHANGES_REQUESTED** [review]
+ I would change the commit message https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716

chore(suite): replace externalLink component by trezorLink component

File: packages/suite/src/views/settings/index.tsx [hunk 3]
```diff
@@ -366,7 +364,7 @@ const Settings = () => {
                                     values={{
                                         version: (
                                             <VersionTooltip content={process.env.COMMITHASH || ''}>
-                                                <VersionLink
```
Comment (tomasklim) [inline packages/suite/src/views/settings/index.tsx:369]: Underline should not be there
Screenshot 2021-09-07 at 14 35 48
Screenshot 2021-09-07 at 14 35 55

**Review by tomasklim — CHANGES_REQUESTED** [review]

**Comment by ravinalamada** [comment]
> * I would change the commit message https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716
> 
> chore(suite): replace externalLink component by trezorLink component

Ok, thanks for the advice

File: packages/suite/src/components/settings/SettingsMenu/index.tsx (hunk 1 above)
Comment (tomasklim) [reply packages/suite/src/components/settings/SettingsMenu/index.tsx:11]: ❌ @ravinalamada Did you test it?

(Do not resolve conversation by yourself)

File: packages/suite/src/views/settings/index.tsx (hunk 3 above)
Comment (tomasklim) [reply packages/suite/src/views/settings/index.tsx:369]: ❌ @ravinalamada There is also version of app in settings where it is still underlined

**Review by tomasklim — CHANGES_REQUESTED** [review]

**Pull request closed without merging**